addons/map_gestures/functions/fnc_transmit.sqf

```diff
@@ -0,0 +1,41 @@
+/*
+ * Author: Dslyecxi, MikeMatrix
+ * Transmit PFH
+ *
+ * Arguments:
+ * 0: Arguments <ARRAY>
+ * 1: PFH ID <NUMBER>
+ *
+ * Return Value:
+ * Return description <TYPE>
+ *
+ * Example:
+ * [[], 2] call ace_map_gestures_fnc_transmit
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+private ["_proximityPlayers", "_ownerID", "_nameSane"];
+
+params ["", "_pfhId"];
+
+if (!GVAR(EnableTransmit) || !visibleMap) exitWith {
+    call FUNC(endTransmit);
```

Comment:
When calling `FUNC(endTransmit)` through releasing the mouse button, it will be called twice. Once from the mouse button and once while exiting the PFH.